src/view.js

```diff
@@ -20,11 +20,14 @@ const ClassOptions = [
   'regionClass',
   'regions',
   'template',
+  'templateAsEl',
   'templateContext',
   'triggers',
   'ui'
 ];
 
+const TEMPLATE_AS_EL = 'TEMPLATE_AS_EL';
```

Comment:
if for some reason this PR does go through, we should look into changing this into a `Symbol()`